KubeArmor/packaging/kubearmor.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=KubeArmor
+
+[Service]
+User=root
+KillMode=process
+EnvironmentFile=/opt/kubearmor/kubearmor.conf
+WorkingDirectory=/opt/kubearmor/
+ExecStart=/usr/local/bin/kubearmor -logPath=${LOG_PATH} -enableKubeArmorHostPolicy=${ENABLE_HOST_POLICY} -enableKubeArmorVm=${ENABLE_KVM} -gRPC=${GRPC}
```

Comment:
We should consider using [Viper](https://github.com/spf13/viper) for configuration management. It can accept config from flags, files and env files, any of them works.